cmdstanpy/stanfit.py

```diff
@@ -1116,6 +1118,36 @@ def optimized_params_dict(self) -> OrderedDict:
         """Returns optimized params as Dict."""
         return OrderedDict(zip(self.column_names, self._mle))
 
+    def stan_variable(self, name: str) -> np.ndarray:
+        """
+        Return a numpy.ndarray which contains the estimates for the
+        for the named Stan program variable where the dimensions of the
+        numpy.ndarray match the shape of the Stan program variable.
+
+        :param name: variable name
+        """
+        if name not in self._metadata.stan_vars_dims:
+            raise ValueError('unknown name: {}'.format(name))
+        col_idxs = list(self._metadata.stan_vars_cols[name])
+        vals = list(self._mle)
+        xs = [vals[x] for x in col_idxs]
+        shape = ()
+        if len(col_idxs) > 0:
+            shape = self._metadata.stan_vars_dims[name]
+        return np.ndarray(
+            shape=shape, buffer=np.array(xs)
+        )
```

Comment:
This should also change to `np.array(xs).reshape(shape)`

Reply:
done!